AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ var props = Properties().apply {
 var dafnyVersion = props.getProperty("dafnyVersion")
 
 group = "software.amazon.cryptography"
-version = "1.2.0"
+version = "1.2.1"
```

Comment:
Are we doing a release for this?

Reply:
hmm, that's a typo and should have been changed